src/locales/en.json

```diff
@@ -151,9 +152,9 @@
   },
 
   "nav": {
-    "category": "Categories",
-    "discovery": "Collecting of my data",
-    "connected": "Connected"
+    "category": "Services",
```

Comment:
could you rename "category" "services ? same for "connected" > "my_services" ? Sorry, my preview comment wasn't clear enough

Reply:
Don't worry, I saw with @poupotte, there were recent changes to use "Providers" 👍